python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -791,10 +806,88 @@ def plotFootprintHist(self, catalog, filenamer, dataId, butler=None, camera=None
         self.AnalysisClass(catalog, catalog["base_Footprint_nPix"], "%s" % shortName, shortName,
                            self.config.analysis, flags=["base_Footprint_nPix_flag"], qMin=0, qMax=3000,
                            labeller=StarGalaxyLabeller(), flagsCat=flagsCat,
-                           ).plotHistogram(filenamer, stats=stats, camera=camera, hscRun=hscRun,
+                           ).plotHistogram(filenamer, stats=stats, camera=camera, ccdList=ccdList,
+                                           tractInfo=tractInfo, patchList=patchList, hscRun=hscRun,
                                            matchRadius=matchRadius, zpLabel=zpLabel,
                                            filterStr=dataId['filter'], uberCalLabel=uberCalLabel)
 
+    def plotPsfFluxSnHists(self, catalog, filenamer, dataId, butler=None, camera=None, ccdList=None,
+                           tractInfo=None, patchList=None, hscRun=None, matchRadius=None, zpLabel=None,
+                           forcedStr=None, uberCalLabel=None, postFix="", flagsCat=None, logPlot=True,
+                           density=True, cumulative=-1):
+        stats = None
```

Comment:
I am going to punt on this one if that's ok with you as it would be silly to have for this one but none of the other similar calls...the planned restructuring is also almost certainly going to obsolete all of these anyway, so putting in the effort on this ticket is just not worth it.